src/pages/invitee/EventInvite.jsx

```diff
@@ -1,17 +1,28 @@
-import React, { useState } from "react";
+import React, { useEffect, useState } from "react";
 import Navbar from "../../components/Navbar";
 import Footer from "../../components/Footer";
 import { useNavigate, useParams } from "react-router-dom/dist";
+import moment from "moment/moment";
+import fetchApi from "../../services/fetchApi";
+import { BASE_URL, EVENT_BY_TOKEN, ADD_PARTICIPANTS } from "../../services/rootEndPoints";
 
 const EventInvite = () => {
+  const [eventData, setEventData] = useState(null);
+  const { get } = fetchApi;
+  const preferredDate = eventData ? eventData.data.host_prefered_time.replace("-", "") : "";
+  const preferredTime = eventData ? moment(preferredDate, "DD-MM-YYYY HH:mm").format("YYYY-MM-DDTHH:mm") : "";
   const [inviteDetails, setInviteDetails] = useState({
     fullname: "",
     email: "",
     preferred_date_time: ""
   });
   const navigate = useNavigate();
   const [declinedInvite, setDeclinedInvite] = useState(false);
-  let { eventId } = useParams();
+  const [resultMsg, setResultMsg] = useState("")
+  let { eventToken } = useParams();
+  const startDate = eventData ? moment(eventData.data.start_date, "DD-MM-YYYY").format("YYYY-MM-DDTHH:mm") : "";
+  const endDate = eventData ? moment(eventData.data.end_date, "DD-MM-YYYY").format("YYYY-MM-DDTHH:mm") : "";
```

Comment:
- [ ] Kindly use the format presented in the API docs which is the same as the one on the get events page. Get the backend to change the format. Allow inconsistencies will affect us. I apologise for inconvinence